### PR TITLE
SSE-2558 Organise views

### PR DESCRIPTION
**Pull request by BorisP-GDS**
Refactor views as per AC in [SSE-2558](https://govukverify.atlassian.net/browse/SSE-2558) story

[SSE-2558]: https://govukverify.atlassian.net/browse/SSE-2558?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: express/src/routes/sign-in.ts [hunk 1]
```diff
@@ -49,29 +52,35 @@ router
         finishSignIn
     );
 
-router.route("/resend-text-code").get(showResendPhoneCodePage).post(showCheckPhonePage);
+router.route("/resend-text-code").get(showResendPhoneCodePage, showCheckPhonePage);
```
Comment (CharlesIC) [inline express/src/routes/sign-in.ts]: Is this correct? I think we need the post

File: express/src/views/layouts/base-form-client.njk [hunk 2]
```diff
@@ -1,5 +1,5 @@
-{% extends "layout/form.njk" %}
+{% extends "layouts/form.njk" %}
```
Comment (CharlesIC) [inline express/src/views/layouts/base-form-client.njk:1]:
```suggestion
{% extends "./form.njk" %}
```

File: express/src/views/layouts/services.njk [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{% extends "layout/base.njk" %}
+{% extends "layouts/base.njk" %}
```
Comment (CharlesIC) [inline express/src/views/layouts/services.njk:1]:
```suggestion
{% extends "./base.njk" %}
```

File: express/src/views/sign-in/enter-email-address.njk [hunk 4]
```diff
@@ -1,4 +1,4 @@
-{% extends "layout/form.njk" %}
+{% extends "../layouts/form.njk" %}
```
Comment (CharlesIC) [inline express/src/views/sign-in/enter-email-address.njk:1]:
```suggestion
{% extends "layouts/form.njk" %}
```

File: express/src/views/sign-in/resend-text-code.njk [hunk 5]
```diff
@@ -6,7 +6,7 @@
 {% block afterForm %}
   <p class="govuk-body">
     If you no longer have access to the phone number linked to your account, you can
-    <a class="govuk-link" href="/test/security-check-change-number">change your phone number</a>.
+    <a class="govuk-link" href="/security-check-change-number">change your phone number</a>.
```
Comment (CharlesIC) [inline express/src/views/sign-in/resend-text-code.njk:9]: This route is in the testing routes

File: express/src/views/test/confirm-phone-number.njk [hunk 6]
```diff
@@ -1,7 +1,7 @@
-{% extends "layout/form.njk" %}
+{% extends "layouts/form.njk" %}
 
 {% set pageTitle = "Confirm phone number" %}
-{% set backLinkPath = "/test/security-check-change-number" %}
+{% set backLinkPath = "/security-check-change-number" %}
```
Comment (CharlesIC) [inline express/src/views/test/confirm-phone-number.njk:4]: This is a testing route

File: express/src/views/test/new-phone-number.njk [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {% extends "common/change-phone-number.njk" %}
 
 {% set pageTitle = "Enter your new mobile phone number" %}
-{% set backLinkPath = "/test/confirm-phone-number" %}
+{% set backLinkPath = "/confirm-phone-number" %}
```
Comment (CharlesIC) [inline express/src/views/test/new-phone-number.njk:4]: This is a testing route

File: express/src/views/common/check-email.njk
Comment (CharlesIC) [inline express/src/views/common/check-email.njk:1]: Rename to `enter-email-code.njk`

File: express/src/views/sign-in/resend-text-code.njk (hunk 5 above)
Comment (BorisP-GDS) [reply express/src/views/sign-in/resend-text-code.njk:9]: I will remove the code, as it is related to phone update during sign-in, which is not needed at the moment.

**Review by CharlesIC — APPROVED** [review]

**Pull request merged**